src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate toDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder sb = new StringBuilder();
+        sb.append(String.format("Report for period %s - %s", dateFrom, dateTo));
+        for (String name : names) {
+            int salary = 0;
+            for (String datum : data) {
+                String[] parts = datum.split(" ");
```

Comment:
please create a constant e.g. DELIMITER or SEPARATOR insead of using hardcoded values